src/app/components/bible-reader/bible-reader.component.css

```
@@ -79,7 +81,11 @@ h6 {
 
   /*margin-right: 10px;  Adds space to the right of the chapter number */
   /* margin-bottom: 10px; Adds space below the chapter number */
-  color: dimgray;
+  color: var(--text-secondary);
+}
+
+::ng-deep .bookSelector .mat-drawer-inner-container {
```

Stylelint (17.12.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)